pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -91,6 +91,14 @@ type ConverterContext struct {
 	MemBalloonStatsPeriod uint
 }
 
+type CriticalSecurebootError struct {
+        Msg string
+}
+
+func (e *CriticalSecurebootError) Error() string { return e.Msg }
+
+const CriticalSecurebootErrorKey = "secureboot setup error"
```

Comment:
This string is displayed when regular EFI boot failed, so when *not* using secureboot. Could probably be reworded.

Reply:
The string is meant to be unique among error messages, between the virt-launcher and virt-handler. 

Is "EFI bootloader missing for insecure boot" better?